components/Cards/Card.tsx

```diff
@@ -29,7 +29,7 @@ export const Card: FC<CardProps> = ({ data, onClick }) => {
 
   return (
     <article className="z-10 h-full w-full rounded-3xl dark:bg-slate-800 dark:text-text-primary dark:shadow-sm bg-light-white">
-      <div className="card-body">
+      <div className="card-body border border-theme-secondary/25 rounded-xl bg-theme-primary-light dark:border dark:border-theme-primary dark:border-opacity-8 dark:bg-theme-tertiary">
```

Comment:
IMO, class `rounded-xl` is not required

Reply:
Alright, I will remove it. Shall i create another pull request after this?

Reply:
Actually i just double checked and if i remove the rounded-xl class the cards aren't rounded anymore.
![Screenshot 2024-08-10 150843](https://github.com/user-attachments/assets/14007d9d-a045-46b1-b65e-755ee5925ac9)

Reply:
No, just push commit removing the class in your branch.
The pull will be updated.

Reply:
I made the cards look like the ones in the category page as requested. If a maintainer asks me to remove it i will. I will keep it for now.

Reply:
There's another class `rounded-3xl` in `<article>` which is responsible for border-radius

Reply:
Alright i will look at it soon. Thank you!

Reply:
Ok i looked at it and i need to write the border classes into the  `<article>`  and change the bg-color for it to work like i want it to. Is this right?.

Reply:
Yes

Reply:
Ok i updated it. This should hopefully be ok now.